src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -179,11 +180,12 @@ public ResponseEntity patchObjection(
     }
 
     @PostMapping("/{objectionId}/attachments")
-    public ResponseEntity<String> uploadAttachmentToObjection (
+    public ResponseEntity uploadAttachmentToObjection (
```

Comment:
API Swagger spec says that the attachment file id is returned from this endpoint so wondering why method signature has changed?